json-rpc/spec.json

```
@@ -1716,6 +1716,11 @@
 									"maxPriorityFeePerGas"
 								],
 								"properties": {
+									"gasPrice": {
+										"title": "transactionGasPrice",
+										"type": "string",
+										"description": "The effective gas price paid by the sender in Wei. For transactions not yet mined, this value should be set equal to the max fee cap per gas. This field is DEPRECATED, please transition to using effectiveGasPrice in the Receipt object going forward."
```

```suggestion
										"description": "The effective gas price paid by the sender in attoeth. For transactions not yet mined, this value should be set equal to the max fee cap per gas. This field is DEPRECATED, please transition to using effectiveGasPrice in the Receipt object going forward."
```
I figure it is worth a shot...

---

haha

---

this could be refactored to change the `oneOf` wrapping it to be an `allOf`